### DISCUSSION TITLE
Ability to shut off all notifications.

### DISCUSSION THREAD

**Discussion by caj411** [diegopvlk/Dosage#49] [Ideas]
I have a LOT of medications and its constantly pinging me with audible and visual notifications.  Its a great app, but its highly disruptive to be constantly inundated by notifications.  I primarily use it for a tracking app for all my medications, not as a reminder app.

**Comment by diegopvlk** [comment]
Just disable that in settings:
![image](https://github.com/diegopvlk/Dosage/assets/25941156/bcd0d8a2-a84e-4715-ba81-2d259e588325)

**Comment by caj411** [comment]
Sorry, how stupid of me.  I was thinking so much about how to do things in the app that I forgot about notifications at the OS level.
